tests/run/analyse_expressions.srctree

```diff
@@ -0,0 +1,44 @@
+# mode: run
+# tag: coverage
```

Comment:
```suggestion
```

I don't think `tag:coverage` is right. I think that's more for things testing Cython's implementation of code coverage rather than for tests designed to increase our coverage.